File: tests/fixtures/rust-syntax-family/host/crates/syntax-core/src/decisions.rs
// decision:0001 keeps the invoice boundary explicit.
pub fn anchored_decision() -> u8 {
    1
}

/* decision:9999 is deliberately orphaned. */
pub fn orphaned_decision() -> u8 {
    2
}

pub const COMMENT_SHAPED_STRING: &str = "// decision:7777";
pub const COMMENT_SHAPED_RAW: &str = r#"/* decision:8888 */"#;

File: tests/fixtures/ruby-syntax-family/host/lib/syntax_family/decisions.rb
# frozen_string_literal: true

module SyntaxFamily
  module Decisions
    # decision:0001 keeps the Ruby syntax boundary explicit.
    def self.anchored_decision
      :anchored
    end

    # decision:9999 is deliberately orphaned.
    def self.orphaned_decision
      :orphaned
    end

    COMMENT_SHAPED_STRING = "# decision:7777"
  end
end
